index.bs

```diff
@@ -356,19 +360,27 @@ even slightly or slowly, without affecting your driving direction.
 
 The orientation vector of the [=Orientation Sensor=], can be calculated the following way:
 
-The gravity vector points towards the earth's core when mostly stationary and as long as the
-device is not in free fall, there is enough vector length to project the magnetic field vector
-onto the ground plane.
+As the [=Accelerometer=] is an <a>inertial-frame sensor</a>, the gravity vector will point
+towards the sky when device is mostly stationary and as long as the device is not in free fall,
+there is enough vector length to project the magnetic field vector onto the ground plane.
 
 Note, this will fail at the magnetic poles as the magnetic field vector will point mostly in
-the same direction as the gravity vector and generally be very unreliable.
+the opposite direction as the gravity vector and generally be very unreliable.
 
 By taking the cross product between the gravity vector (see [=Gravity Sensor=]) and the magnetic
-field vector, we get a vector which points East on the ground plane, using the right hand rule.
+field vector, we get a vector which points West on the ground plane, using the right hand rule.
```

Comment:
But this is not about coordinate system, usually you take uG x uB and then again cross product with gravity vector. Since the gravity vector points upwards, cross product would be uW that points west.

Developers might be confused if they read doc and it says "gravity vector points towards earth", yet device reports different values. I tried to add example that is based on what sensors provide.

@kenchris wdyt?